src/metamath.c

```diff
@@ -3700,150 +3682,150 @@ void command(int argc, char *argv[]) {
 
           if (switchPos("COMPRESSED")) {
             let(&str1, compressProof(nmbrSaveProof,
-                outStatement, /* g_showStatement or g_proveStatement based on pipFlag */
+                outStatement, // g_showStatement or g_proveStatement based on pipFlag
                 (switchPos("OLD_COMPRESSION")) ? 1 : 0));
           } else {
             let(&str1, nmbrCvtRToVString(nmbrSaveProof,
                 explicitTargets,
-                outStatement /*statemNum, used only if explicitTargets*/));
+                outStatement)); // statemNum, used only if explicitTargets
           }
 
           if (saveFlag) {
-            /* ??? This is a problem when mixing html and save proof */
+            // ??? This is a problem when mixing html and save proof
             if (g_printString[0]) bug(1114);
             let(&g_printString, "");
 
-            /* Set flag for print2() to put output in g_printString instead
-               of displaying it on the screen */
+            // Set flag for print2() to put output in g_printString instead
+            // of displaying it on the screen.
             g_outputToString = 1;
           } else {
             if (!print2("Proof of \"%s\":\n", g_Statement[outStatement].labelName))
-              break; /* Break for speedup if user quit */
+              break; // Break for speedup if user quit
             print2(
 "---------Clip out the proof below this line to put it in the source file:\n");
           }
           if (switchPos("COMPRESSED")) {
             printLongLine(cat(space(indentation), str1, " $.", NULL),
-              space(indentation), "& "); /* "&" is special flag to break
-                  compressed part of proof anywhere */
+              // "&" is special flag to break compressed part of proof anywhere.
```

Comment:
this line should move up one line (can't seem to use suggestion)

Reply:
Done in 02aa6e9